sql/pr/avg-time-to-first-review.sql

```diff
@@ -0,0 +1,21 @@
+-- Average time to first review
+WITH first_reviews AS (
+    SELECT
+        pulls.number as pull_number,
+        pulls.created_at as created_at,
+        min(reviews.submitted_at) as first_review
+    FROM pulls
```

Comment:
Use the view `unique_pulls` instead of the `pulls` table, because we keep inserting updated PRs there.